src/plugins/kibana_react/public/empty_state_page/empty_state_page.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { EuiPageTemplate } from '@elastic/eui';
+import React from 'react';
+import { KibanaPageTemplateProps, NO_DATA_PAGE_TEMPLATE_PROPS, NoDataPage } from '../page_template';
+
+interface Props {
+  classes: string;
+}
+
+type EmptyPageProps = Props & KibanaPageTemplateProps;
+
+export const EmptyStatePage = (props: EmptyPageProps) => {
+  const { template, classes, pageSideBar, pageSideBarProps, noDataConfig } = props;
+
+  if (!noDataConfig) {
+    return null;
+  }
+
+  return (
+    <EuiPageTemplate
+      data-test-subj={props['data-test-subj']}
+      template={template}
+      className={classes}
```

Comment:
is this a typo?  Should it be `classes={classes}`...?

Reply:
`className` is the correct prop name for all EUI components as it extends from React's own API. I know there's been some skepticism lately in the community around this name, but as it's ingrained in all of EUI and known throughout Elastic's engineers, I would highly recommend sticking with `className` over anything else. So in fact, this top level component should change it's props from `classes` to `className`.